app/Models/Post.php

```diff
@@ -203,12 +203,22 @@ public function untag($tagName)
     }
 
     /**
-     * Remove the given tag from this post.
+     * Returns posts without specific tag.
      */
     public function scopeWithoutTag($query, $tagSlug)
     {
         return $query->whereDoesntHave('tags', function ($query) use ($tagSlug) {
             $query->where('tag_slug', '=', $tagSlug);
         });
     }
+
+    /**
+     * Returns posts with specific tag.
+     */
+    public function scopeWithTag($query, $tagSlug)
```

Comment:
Nice!

Reply:
Should we have it be `withTags()` and instead of checking for equality, we check if the `tag_slug` is in a list of tags?

Reply:
I was thinking that too and like that idea! For filtering by tag, you can only filter a tag one at a time (from the drop down menu) so we don't need it for this but could be good for the api in general to be able to send in multiple tags!

Reply:
Yea and it looks like we only allow you to filter out one tag. so I think both of these should be updated. Will make a card!

Reply:
awesome, thanks Shae!